tests/karma/unit/controllers/tasks.js

```diff
@@ -71,7 +81,8 @@ describe('TasksCtrl controller', function() {
       TaskGenerator.args[0][1](null, expected);
       chai.expect(TaskGenerator.callCount).to.equal(1);
       chai.expect(scope.filterModel.type).to.equal('tasks');
-      chai.expect(scope.tasks).to.deep.equal(expected);
+      chai.assert.isTrue(LiveList.tasks.update.calledWith(expected[0]));
+      chai.assert.isTrue(LiveList.tasks.update.calledWith(expected[1]));
```

Comment:
It'd be nice to stay consistent with the surrounding code and use expect or assert form but not both...